README.md

```diff
@@ -259,6 +259,7 @@ step to anywhere after the MoveGranule step being sure to change the
                   "distribution_endpoint":"{$.meta.distribution_endpoint}",
                   "files_config":"{$.meta.collection.files}",
                   "multipart_chunksize_mb": "{$.meta.collection.multipart_chunksize_mb"},
+                  "orcaDefaultBucketOverride": "{$.meta.collection.meta.orca_default_bucket}"
```

Comment:
Need to change `"orcaDefaultBucketOverride": "{$.meta.collection.meta.orca_default_bucket}"` to 
`"orcaDefaultBucketOverride": "{$.meta.collection.meta.orcaDefaultBucketOverride}"` to be consistent with our other items in collection configuration.

Reply:
Fixed